tests/References/ResolverTest.php

```diff
@@ -120,4 +129,23 @@ public function testUnResolvedReference2(): void
 
         self::assertNull($this->resolver->resolve($this->environment, 'invalid-reference'));
     }
+
+    public function testMissingReferenceResolverEventDispatched(): void
+    {
+        $this->environment->expects(self::any())
```

Comment:
This is a no-op, you can omit it. 

```suggestion
        $this->environment
```